translator/trace/spandata/protospan_to_spandata.go

```diff
@@ -47,8 +47,8 @@ func ProtoSpanToOCSpanData(span *tracepb.Span) (*trace.SpanData, error) {
 	sd := &trace.SpanData{
 		SpanContext:     sc,
 		ParentSpanID:    parentSpanID,
-		StartTime:       timestampToTime(span.StartTime),
-		EndTime:         timestampToTime(span.EndTime),
+		StartTime:       internal.TimestampToTime(span.StartTime),
```

Comment:
I missed this earlier but `internal` is not a good name for a package.

Reply:
This is not a new package. We can rename it to something better in an future PR.